src/controller/AutoCommissioner.h

```diff
@@ -113,6 +108,17 @@ class AutoCommissioner : public CommissioningDelegate
     void ResetTryingSecondaryNetwork() { mTryingSecondaryNetwork = false; }
     bool mTryingSecondaryNetwork = false;
 
+    CHIP_ERROR AllocateMemoryAndCopySpan(Platform::ScopedMemoryBufferWithSize<uint8_t> & scopedBuffer, ByteSpan span)
```

Comment:
Why is this inlined?  I'd think we want this in the .cpp file.

Reply:
moved it in the .cpp file